EIPS/eip-4841.md

```diff
@@ -0,0 +1,418 @@
+---
+eip: 4841
+title: Expandable Onchain SVG Images Storage
+description: It is a Expandable Onchain SVG Images Storage Structure Model on Ethereum.
```

Comment:
Description should not start with "it is" - the subject is implied.

Reply:
Done! Thanks for your review.